compute/containers/api-cli/restricting-access-to-a-container.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Restricting access to a function
+description: This page explains how to restrict access to a function
+excerpt: |
+  This page explains how to restrict access to a function
+keywords: cloud computing, scaleway, serverless, functions, namespace, manage
+dates:
+  validation: 2021-05-26
+  posted: 2021-05-26
+---
+
+This page shows you how to restrict access to your container by making it private, thus preventing unauthorized launching of it. Contrary to private dunctions, private containers need to be configured to prevent unauthenticated requests to access the container.
```

Comment:
```suggestion
This page shows you how to restrict access to your container by making it private. This prevents unauthorized launching of the container. Contrary to private functions, private containers need to be configured to prevent unauthenticated requests to access the container.
```